### PR TITLE
Add review suggestions by James M. Croft

### PR DESCRIPTION
**Pull request by itrich**
Thanks to the kind offer by [Cloudical](https://cloudical.io/), our manifesto was reviewed by a native speaker who made the proposed changes.

Feel free to comment, adjust or review!

Kudos to @FriederikeZelke for organizing this opportunity! :heart:

### CODE REVIEW

File: MANIFESTO.md [hunk 1]
```diff
@@ -3,37 +3,37 @@ layout: page
 permalink: /
 image: assets/openops.png
 ---
-We – [the founding and supporting organizations](#supporting-organizations) – proclaim transparency, sharing knowledge and are building a community of practice - open operations.
+We – [the founding and supporting organizations](#supporting-organizations) – proclaim our primary objectives to be transparency and the sharing of knowledge, and are in the process of building a community of practice – Open Operations.
```
Comment (maxwolfs) [inline MANIFESTO.md:6]: As german native speaker, two "ands" within an enumeration of three sounds strange but if it's proper english okay. 🤔

Comment (itrich) [reply MANIFESTO.md:6]: We could replace it with **to be transparency as well as the sharing of knowledge**

File: MANIFESTO.md [hunk 2]
```diff
@@ -3,37 +3,37 @@ layout: page
...
+We firmly believe in the princple of learning from our mistakes and that lessons learnt add to our expertise. The way we handle errors is how we become better. We feel that handling our mistakes in the best way possible means communicating about them openly and without shame. We are convinced that the culture we build around failures is fundamental for an innovative environment: not being afraid of making mistakes opens the door to trying out new approaches, without fear of failing. We encourage people to speak out – not just when they are successful, but also when they fail. We are sure that this is indeed one of the best ways to prevent ourselves and others from making the same mistakes. We hold that to achieve the state of complete transparency, the history of failures and incidents should be recorded and preserved. Our goal is that others can truly experience trust in this way, rather than just being told they can.
```
Comment (fkr) [inline MANIFESTO.md:22]: I'm saddened to see the sentence "failures make us experts" being removed and fail to see the reason why this - in my opinion - important way of stating something in a very explicit manor needs to go.

File: MANIFESTO.md [hunk 3]
```diff
@@ -3,37 +3,37 @@ layout: page
...
+in other, more innovative ways. Open Operations is building a community of practice to keep the barrier to market entry low and create a thriving environment for easy and convenient exchange. Regardless of the individual level of knowledge, everyone is encouraged to contribute and thereby enriches this community of practice with their knowledge and individual views. Communities of practice connect through various channels, serving both asynchronous and synchronous communication needs in the form of pair operating or on a larger scale to solve these challenges cooperatively. We are hence building a network by encouraging organizations to foster and connect internal and external communities of practice.
```
[the remark this answers is not in the document]
Comment (itrich) [reply MANIFESTO.md]: Sorry, that was an issue on my side. Fixed this

**Review by fkr — COMMENTED** [review]
Thanks for the good rework. In almost all places this feels really good.

In at least one case it actually changes our wording in more than a nitty bitty way.
If the majority here thinks it is still our wording I'm fine with it. 
Especially with the removal of 'failures make us experts' my wording takes a strong hit.

File: MANIFESTO.md (hunk 2 above)
Comment (itrich) [reply MANIFESTO.md:22]: We are not obliged to accept all changes. Maybe we just stick to the "failures makes us experts" wording (which I do like). Or maybe just ask James why he proposed this change?

Comment (fkr) [reply MANIFESTO.md]: @itrich please check the change from 84af578. Maybe this is a good way?

File: MANIFESTO.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ The availability of knowledge and skilled engineers is the limiting factor for m
...
+We firmly believe that failures make us experts and in the princple of learning from our mistakes and thus lessons learnt add to our expertise. The way we handle errors is how we become better. We feel that handling our mistakes in the best way possible means communicating about them openly and without shame. We are convinced that the culture we build around failures is fundamental for an innovative environment: not being afraid of making mistakes opens the door to trying out new approaches, without fear of failing. We encourage people to speak out – not just when they are successful, but also when they fail. We are sure that this is indeed one of the best ways to prevent ourselves and others from making the same mistakes. We hold that to achieve the state of complete transparency, the history of failures and incidents should be recorded and preserved. Our goal is that others can truly experience trust in this way, rather than just being told they can.
```
Comment (itrich) [inline MANIFESTO.md:22]: What about:

We firmly believe that failures make us experts. In the principle of learning from our mistakes the lessons learnt add to our expertise. Thus, the way we handle...

Comment (fkr) [reply MANIFESTO.md:22]: ack

**Review by maxwolfs — APPROVED** [review]
LGTM! Let's give it a go!

**Pull request merged**